en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+For example the byy far most common battery type for remote controlled vehicles are lithium polymer (li-po) accumulators with a nominal cell voltage of 3.7V.
+To achieve higher voltages for efficiently powering the actuators of a vehicle, multiple cells are connected in series.
+This results in multiples of the cell voltage as output at the battery terminals. Batteries are labeled with the number of cells they contain by a number followed by a big S for series. E.g. 1S is one cell, 4S are four cells in series.
+If you cannot find any label on your battery telling you the number of cells you can infer the count by the battery type and its nominal voltage.
+
+So for li-po batteries the following voltages are possible
+* 1S - 3.7V
+* 2S - 7.4V
+* 3S - 11.1V
+* 4S - 14.8V
+* 5S - 18.5V
+* 6S - 22.2V
+
+This setting sets the number of cells connected in series PX4 should monitor.
+
+### Full cell voltage
+A chemical battery cells output voltage depends on many parameters like the way the battery was built, temperature, load, if the battery was damaged. But for a healthy battery under normal conditions with no load attached the remaining capacity is the main factor.
+Normal li-po batteries should be charged by dedicated charging devices such that each cell reaches a maximal voltage of exactly 4.2V. But after the battery was fully charged and is stored for later use the voltage starts to decrease even with no external load at all.
+That's why this setting should be set just a bit lower than the maximal voltage of a cell otherwise the fuel estimate will likely already be below 100% by the time you connect the battery to your autopilot.
+
+### Empty cell voltage
+Most battery cells particularly also li-pos suffer a lot when they get used after they are "empty". To say if they are empty is not binary and not accurately possible by only knowing the cell voltage.
+On one hand if a li-po ever sees a voltage lower than 3.2V even if it's under load it will suffer significant damage for sure. On the other hand the nominal cell voltage 3.7V without load is where the battery starts to get into the empty range and that's the most conservative voltage to consider it empty.
+In between these extremes the earlier you put the battery to charge again the longer the life span will be and the less capacity degrade you'll see.
+
+> **Hint** As my personal most general rule of thumb if the first li-po cell raw voltage gets below 3.5V during flight (under load) I start to land immediately.
+
+When speaking about the empty cell voltage we need to distinguish:
+* Raw voltage under load, the voltage directly measured while flying
+This value highly varies depending on the momentary current draw from the cells. As a result you will see fluctuating battery percentages
+* Rest voltage without a load when the battery is resting for some time
+Best suitable measurement to infer charge status of the cell.
+* Load compensated voltage calculated in air (under load)
```

Comment:
Here you introduce "load compensation" but I know nothing about this, and neither will your reader. Is this something that is provided by a battery or by PX4 or by a power module. How would we know if this is enabled?

Reply:
Simplest explanation: The load compensated voltage is the voltage the battery would currently have without load but you are trying to guess it during operation and hence load. This is useful because the current discharge level can better be infered from a battery voltage when it is not under load.

PX4 uses a load compensated voltage value for its battery voltage estimation if configured to do so. The voltage you see in the UI and log is always the real measured voltage and never load compensated.

- If you set the parameter `BAT_V_LOAD_DROP` it uses compensation based on the current total motor thrust which is an inacurate way of guessing the voltage without load because you assume the motor commands directly and instantaneously translate to electrical load.
- If you set `BAT_R_INTERNAL` it uses compensation based on the current measurement which basically assumes an [elecrical internal resistance model](https://en.wikipedia.org/wiki/Internal_resistance) that is more accurate.